src/components/Loader/styled.tsx

```diff
@@ -0,0 +1,33 @@
+import styled, { keyframes } from 'styled-components/macro'
+
+export const loadingAnimation = keyframes`
+  0% {
+    background-position: 100% 50%;
+  }
+  100% {
+    background-position: 0% 50%;
+  }
+`
+
+export const LoadingRows = styled.div`
+  display: grid;
+
+  & > div {
+    animation: ${loadingAnimation} 1.5s infinite;
+    animation-fill-mode: both;
+    background: linear-gradient(
+      to left,
+      ${({ theme }) => theme.bg1} 25%,
+      ${({ theme }) => theme.bg2} 50%,
+      ${({ theme }) => theme.bg1} 75%
+    );
+    background-size: 400%;
+    border-radius: 12px;
+    height: 2.4em;
+    will-change: background-position;
+  }
+`
+
+export function opacityLoadingStyles(loading: boolean) {
+  return { transition: 'opacity 0.2s ease-in-out', ...(loading ? { opacity: '0.3' } : { opacity: '1' }) }
+}
```

Comment:
pref doing this with styled-components instead of inline styles https://styled-components.com/docs/api#css

Reply:
neat! so much better